saleor/graphql/account/mutations.py

```diff
@@ -642,3 +659,58 @@ def mutate(cls, root, info, id, type):
 
         utils.change_user_default_address(user, address, address_type)
         return cls(errors=errors, user=user)
+
+
+class UserImageUpdate(BaseMutation):
+    user = graphene.Field(User, description='An updated user instance.')
+
+    class Arguments:
+        image = Upload(
+            required=True,
+            description='Represents an image file in a multipart request.',
+        )
+
+    class Meta:
+        description = dedent(
+            '''
+            Create a user image. This mutation must be
```

Comment:
I would also say in the description that this mutation applies only to staff members. We have a permission check but it would be nice to tell that in the description as well. The same for the "delete" mutation.